index.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!--the nav-bar -->
+    <section id="the-nav-cont" class="the-general-cont">
+      <div class="navvv-cont">
+        <a href="#" class="logo">Mahan Sanjarani</a>
+        <nav class="navvbar">
+          <a href="#" class="opt">
+            <img src="./img/menu/Icon-Menu.png" alt="option icon" />
+          </a>
+          <a href="" class="n-b-list">Portfolio</a>
+          <a href="#about" class="n-b-list">About</a>
+          <a href="#contact" class="n-b-list">Contact</a>
+          <a href="" class="n-b-list-icon"
+            ><img src="./img/menu/Mail-icon.png" alt="Mail"
+          /></a>
+        </nav>
+      </div>
+    </section>
```

Comment:
[OPTIONAL] Since this is the header, consider wrapping it in the header tag instead of section tag. :pray: